apps/dokploy/server/api/routers/application.ts

```diff
@@ -112,7 +113,16 @@ export const applicationRouter = createTRPCRouter({
 					message: "You are not authorized to access this application",
 				});
 			}
-			return application;
+			const {
+				password,
+				...applicationWithoutPassword
+			}: Application & {
+				passwordLength?: number;
+			} = application;
+			applicationWithoutPassword.passwordLength =
+				application.password?.length || 0;
+
+			return applicationWithoutPassword;
```

Comment:
To prevent errors do this:


```suggestion
						const {
				password,
				...rest
			}: Awaited<ReturnType<typeof findApplicationById>> & {
				passwordLength?: number;
			} = application;

			rest.passwordLength = application.password?.length || 0;

			return rest;
```